src/plugins/kibana/public/settings/sections/advanced/index.html

```diff
@@ -3,8 +3,7 @@
     <div class="bs-callout bs-callout-warning">
       <h4>Caution: You can break stuff here</h4>
       Be careful in here, these settings are for very advanced users only. Tweaks you make here can break large portions of Kibana.
-      Some of these settings may be undocumented, unsupported or experimental. Blanking a field will cause Kibana to use its internal
-      defaults which may be unacceptable given other configuration directives.
+      Some of these settings may be undocumented, unsupported or experimental. If a field has a default value, blanking the field will reset it to its default which may be unacceptable given other configuration directives. Deleting a custom setting will permanently remove it from Kibana's config.
```

Comment:
Mind putting some new lines in here? We don't really enforce line length in HTML, but newlines in HTML text don't affect the result either.